perftest/simulations/src/gatling/scala/org/projectnessie/perftest/gatling/MixedWorkloadsParams.scala

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright (C) 2020 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.perftest.gatling
+
+import io.gatling.core.session.Session
+import java.lang.Integer.getInteger
+import java.lang.System.getProperty
+import java.util.concurrent.ThreadLocalRandom
+import org.projectnessie.model.{ContentKey, Namespace}
+import scala.concurrent.duration.{Duration, FiniteDuration, HOURS, NANOSECONDS}
+
+case class SessionsAndRate(
+    users: Int,
+    opRate: Double
+) {
+  def rateToDuration(): FiniteDuration = {
+    val oneHour = FiniteDuration(1, HOURS)
+    val nanosPerIteration = oneHour.toNanos / (opRate * oneHour.toSeconds)
+    FiniteDuration(nanosPerIteration.toLong, NANOSECONDS)
+  }
+
+  override def toString: String = s"$users users at rate $opRate"
+}
+
+object SessionsAndRate {
+  def fromSystemProperties(
+      prefix: String,
+      usersDefault: Int = 1,
+      opRateDefault: Double = 10
+  ): SessionsAndRate = {
+    val users: Int = getInteger(s"$prefix.users", usersDefault)
+    val rate: Double = getProperty(s"$prefix.rate", s"$opRateDefault").toDouble
+    SessionsAndRate(users, rate)
+  }
+}
+
+case class MixedWorkloadsTables(
+    namespaceStr: String,
+    totalTables: Int,
+    activeTables: Int,
+    suffix: String = ""
+) {
+  def contentKey(t: Int, session: Session): ContentKey = {
+    val namespace = session("namespace").as[Namespace]
+    ContentKey.of(namespace, s"table-${t}$suffix")
+  }
+
+  def randomActiveTable(): Int = {
+    ThreadLocalRandom.current().nextInt(activeTables)
```

Comment:
This will always cause active keys to be in front (lexicographically) of the total list of tables. Would it be preferable to choose a random but deterministic sub-set of active tables spanning the whole list?

Reply:
Maybe as a follow-up

Reply:
sure